india_banking_connector/connectors/doctype/axis_bank_connector/axis_bank_connector.py

```
@@ -0,0 +1,396 @@
+		if method in method_map:
+			method_map[method](decrypted_data, res_dict)
+
+	def get_payment_status(self):
+		payment_details = self.doc
+		unique_id = "".join(re.findall(r"[0-9a-zA-Z]", payment_details.name))
+
+		url = self.urls.payment_status
+		headers = self.headers
+		payload = self.get_encrypted_payload(method="payment_status")
+
+		response = requests.post(
```

Ruff (0.15.1)

[error] Probable use of `requests` call without timeout

(S113)